packages/ag-charts-community/src/module-support.ts

```diff
@@ -38,6 +38,7 @@ export * from './chart/interaction/interactionManager';
 export * from './chart/interaction/gestureDetector';
 export * from './chart/interaction/tooltipManager';
 export * from './chart/interaction/zoomManager';
+export { type Chart } from './chart/chart';
```

Comment:
I don't like exporting `Chart`, as it's interface contract is a bit too wild and unrestricted - could we introduced a narrower interface contract for sync-related methods which `Chart` implements?